src/Autocomplete/src/AutocompleteResultsExecutor.php

```diff
@@ -71,7 +71,9 @@ public function fetchResults(EntityAutocompleterInterface $autocompleter, string
 
         $results = [];
 
-        if (null === $groupBy = $autocompleter->getGroupBy()) {
+        if (!method_exists($autocompleter, 'getGroupBy')) {
+            trigger_deprecation('symfony/ux-autocomplete', '2.8', 'Not implementing the "EntityAutocompleterInterface::getGroupBy()" in "%s" is deprecated.', get_debug_type($autocompleter));
```

Comment:
This deprecation is not needed, the `@method` annotation will already trigger a similar deprecation (which is done via Symfony's `DebugClassLoader`).

Reply:
thx - got it at https://github.com/symfony/ux/pull/808